libipt/internal/include/pt_image.h

```diff
@@ -80,7 +80,7 @@ extern void pt_image_fini(struct pt_image *image);
  *
  * Add @section identified by @isid to @image at @vaddr in @asid.  If @section
  * overlaps with existing sections, the existing sections are shrunk, split, or
- * removed to accomodate @section.  Absence of a section identifier is indicated
+ * removed to accommodate @section.  Absence of a section identifier is indicated
```

Comment:
This moves the line beyond the 80 columns limit.  Please change the formatting to stay within 80 columns.